sandy_sort_lists.py

```diff
@@ -0,0 +1,94 @@
+import config
+import requests
+import json
+
+
+COLOR_KEYS = {
+	"green": 1,
+	"yellow": 2,
+	"orange": 3,
+	"red": 4,
+	"purple": 5,
+	"blue": 6,
+	"sky": 7,
+	"lime": 8,
+	"pink": 9,
+	"black": 10
+}
+BOARD_IDS_URL = "https://api.trello.com/1/members/{username}/boards"
+GET_LISTS_URL = "https://api.trello.com/1/boards/{board_id}/lists"
+GET_CARDS_URL = "https://api.trello.com/1/lists/{list_id}/cards"
+PUT_CARDS_URL = "https://api.trello.com/1/cards/{card_id}/pos"
+BASIC_PARAMS = {
+        "key": config.TRELLO_APP_KEY,
+        "token": config.TRELLO_APP_TOKEN
+        }
+
+
+def get_key_value(card):
+	try:
+		color = COLOR_KEYS[card["labels"][0]["color"]]
+	except IndexError, e:
+		color = 0
+	return color
+
+
+def get_lists_from_board(board_id):
```

Comment:
Ideas:

* What if I don't want to sort each list in the board? E.g. my Done list. I don't want to sort that!
* What if we could specify list ids in the config instead?